### ISSUE TITLE
[SUGGESTION] Make Holo-projectors projections scalable through the slider brick

### ISSUE THREAD

**Issue by Womble-TC** [tsunamayo/Starship-EVO#3328]
I think it would be a good idea to make the projection from the holoprojector be scalable with a slider, like the picture shown below:  
![image](https://user-images.githubusercontent.com/29961381/100683743-e0743d00-3370-11eb-88bd-5685c43dda81.png)
That's all I really have to say.

**Comment by BrickMacklin** [comment]
I could personally use an enlarged galaxy map here
![image](https://user-images.githubusercontent.com/37836252/100684231-eace1100-3347-11eb-8bc1-07d6ee93d515.png)

**Comment by ProPeach** [comment]
This functionality would be fantastic, although personally I think a live updating slider in an [F] GUI would be more intuitive and could offer other options like offsetting the hologram vertically etc

**Comment by tsunamayo** [comment]
Yes guys it is planned, it will be a small slider inside the holomap menu. Thanks

**Comment by BrickMacklin** [comment]
Thanks @tsunamayo !

**Comment by Garrett-C** [comment]
> This functionality would be fantastic, although personally I think a live updating slider in an [F] GUI would be more intuitive and could offer other options like offsetting the hologram vertically etc

It would be really nice to have a height offset yeah.
